plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeModule.java

```diff
@@ -92,6 +94,7 @@ public void setup(Binder binder)
         configBinder(binder).bindConfig(DeltaLakeConfig.class);
         configBinder(binder).bindConfig(HiveConfig.class);
         binder.bind(MetastoreConfig.class).toInstance(new MetastoreConfig()); // currently not configurable
+        newOptionalBinder(binder, Key.get(boolean.class, TranslateHiveViews.class));
```

Comment:
Looks like extension point, but Delta doesn't need to create extension points. At best, Hive would do this.

what about

```java
binder.bind(Key.get(boolean.class, TranslateHiveViews.class)).toInstance(false);
```

Reply:
tough I guess it is not needed for delta lake as there is still hive config there